src/openeo_test_suite/lib/compliance_util.py

```diff
@@ -380,7 +381,7 @@ def post_jobs(base_url: str, bearer_token: str):
 
         response = requests.post(
             full_endpoint_url,
-            data=payload,
+            data=json.dumps(payload),
```

Comment:
in requests, I think you can just do this with argument `json=payload` instead of manually JSON-encoding `data` argument

Reply:
This is actually needed.